### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by SimpleBinary** [Yepicsend/Yepicsend#3]
Got a question that you really want to be answered? Notify one of our team members and we will try and answer as soon as possible.

**Comment by BonfireScratch** [comment]
@Dandigit what is the team's priority at the moment?

**Comment by Dandigit** [comment]
@BonfireScratch I'm reading up on OAuth, a common technique for API authentication, so I can implement access security.

**Comment by BonfireScratch** [comment]
@Dandigit interesting! OAuth could be of great use in one of my projects.

**Comment by BonfireScratch** [comment]
@Dandigit do you use REST for the API?

**Comment by Dandigit** [comment]
@BonfireScratch Yes, we're using a REST architecture.

**Comment by BonfireScratch** [comment]
@Dandigit does MongoDB need a Node server?

**Comment by SimpleBinary** [comment]
@BonfireScratch No, you can use MongoDB without a server, you can even use it without node.js in languages like Python.

**Comment by BonfireScratch** [comment]
@SimpleBinary I'm talking about JavaScript though, does this make a difference?

**Comment by SimpleBinary** [comment]
@BonfireScratch Do you mean locally? Not locally you need a server.

**Comment by BonfireScratch** [comment]
@SimpleBinary yes. If you have a server, how can you buy a domain name (using namecheap.com for instance)?

**Comment by SimpleBinary** [comment]
@BonfireScratch You can buy a domain name!

**Comment by BonfireScratch** [comment]
@SimpleBinary I mean, how do you make the domain name use the server?

**Comment by Dandigit** [comment]
@BonfireScratch You direct the domain name's A/AAAA DNS records to the server's IP address.

**Comment by BonfireScratch** [comment]
@SimpleBinary @Dandigit how do I setup a server, so that I can code in PHP or Node.js? (my father has a server in a Rapsberry Pi, but I don't know how to use it)

**Comment by Dandigit** [comment]
@BonfireScratch There's an abundance of articles on the web for this kind of thing.
